api/app/jobs/viirs_snow.py

```diff
@@ -233,7 +235,7 @@ async def _run_viirs_snow(self):
         today = date.today()
         if last_processed_date is None:
             # Case to cover the initial run of VIIRS snow processing (ie. start processing one week ago)
-            next_date = today - timedelta(days=10)
+            next_date = today - timedelta(days=21)
```

Comment:
The last snow processed in prod was from Feb 23. Allowing the job to start up to 3 weeks in the past to catch up using version 1 data.